app.js

```diff
@@ -5,70 +5,16 @@ const source = require('rfr');
 const helmet = require('helmet');
 const mongoose = require('mongoose');
 
-const Bot = source('lib/bot');
-const { getSpotifyAuthToken } = source('lib/spotify/spotifyApi');
 const logger = source('lib/utils/logger');
+const router = source('routes');
 
 const app = express();
 
 app.use(helmet());
-
 app.engine('handlebars', exphbs());
 app.set('view engine', 'handlebars');
-
-app.use((req, res, next) => {
-    logger.http(`${req.method} ${req.originalUrl} ${req.ip}`);
-    next();
-});
-
 app.use(express.static('static'));
-
-// home page so you can see that this is running
-app.get('/', (req, res) => {
-    res.render('home', {
-        clientID: Bot.client.user.id,
-    });
-});
-
-app.get('/documentation', (req, res) => {
-    res.render('docs');
-});
-
-// So Kaffeine can ping the application
-app.get('/status', (req, res) => {
-    logger.info('status ping');
-    res.status(204).end(); // no content but status okay
-});
-
-app.get('/authorize', (req, res) => {
-    const userid = req.query.state;
-    const { code } = req.query;
-    let error = req.query.error || 'Missing the userid and the authentication code';
-
-    if (userid && code) {
-        getSpotifyAuthToken(userid, code);
-        error = null;
-    }
-
-    return res.render('authenticate', {
-        error,
-    });
-});
-
-// 404 Errors
-app.use((req, res) => {
-    res.render('404');
-});
-
-let port = process.env.PORT;
-if (port == null || port === '') {
-    port = 8300;
-}
-
-app.listen(port, () => {
-    Bot.client.login(process.env.DISCORD_TOKEN);
-    logger.info(`Server started on port ${port}`);
-});
+app.use(router);
 
 mongoose.connect(process.env.DATABASE_URL, {
```

Comment:
Should this stuff be moved into the `server.js` file as well?

Reply:
the mongoose.connect or the router?

Reply:
the mongoose connect part.

Reply:
I mean it could go in either files and still work. Right now, all server.js is responsible for is starting the app to listening in to a port, app.js is responsible for the app. I think it make more sense for mongoose to be in app.js

Reply:
right now `sever.js` is responsible for starting up the discord bot and starting up and running the webserver, `app.js` is responsible for setting the webapp route handlers and connecting to the db. If the discord bot part had been in the `app.js` file I would agree.

Reply:
Okay, im just gonna remove the server.js file, and use app.js instead